linkerd/datadog_checks/linkerd/linkerd.py

```diff
@@ -21,3 +21,7 @@ def __init__(self, name, init_config, agentConfig, instances=None):
             'linkerd': {'labels_mapper': labels_mapper, 'metrics': [METRIC_MAP], 'type_overrides': TYPE_OVERRIDES}
         }
         super(LinkerdCheck, self).__init__(name, init_config, agentConfig, instances, default_config, 'linkerd')
+
+    def service_check(self, name, status, tags=None, hostname=None, message=None):
```

Comment:
Let's instead overload the check method since https://github.com/DataDog/integrations-core/blob/dd0b53a220e472509d50602a0a065a1503a99f22/datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py#L433 can be disabled

Reply:
I did process since it has the data I needed.

Reply:
Sorry, what I meant was let's make a new service check so the health one can still be disabled. That would be a breaking change.